app/views/compute_resources/form/_ovirt.html.erb

```diff
@@ -1,6 +1,7 @@
 <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
 <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
 <%= password_f f, :password, :unset => unset_password? %>
+<%= checkbox_f f, :filtered_api, :help_inline => _("If you don't have admin permissions you can use slower filtered API."), :label => _('User level API') %>
```

Comment:
I can express again my thoughts on not having this support... I think it
shouldn't be added.
It will not be slower. not in every use-case.
I think it is weird to write that down at all.
It is not filtered API, it is our user-level API.
I think you're getting yourself into mess by adding this.


On Mon, Mar 21, 2016 at 10:45 AM, Lukáš Zapletal <notifications@github.com>
wrote:

> In app/views/compute_resources/form/_ovirt.html.erb
> <https://github.com/theforeman/foreman/pull/3288#discussion_r56792403>:
>
> > @@ -1,6 +1,7 @@
> >  <%= text_f f, :url, :size => "col-md-8", :help_block => _("e.g. https://ovirt.example.com/api"), :help_inline => documentation_button('5.2.7oVirt/RHEVNotes') %>
> >  <%= text_f f, :user, :help_block => _("e.g. admin@internal") %>
> >  <%= password_f f, :password, :unset => unset_password? %>
> > +<%= checkbox_f f, :filtered_api, :help_inline => _("If you don't have admin permissions you can use slower filtered API."), :label => _('User level API') %>
>
> I would go with whatever terminology is used in the oVirt documentation.
> Out of curiosity, can you send a link to the doc that describes this term?
> Because to me, it looks like there is no oVirt documentation at all. ;-)
> … <#-1306898184106590848_>
> -- Later, Lukas #lzap Zapletal
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/theforeman/foreman/pull/3288/files/a8b18e0690cea4dce35e07dc7a427bff0e072036#r56792403>
>